services/wiki/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.16.1] - 2024-05-23
```

Comment:
@kevinmamaqi Since we've removed the GET banners endpoint, should this be a major version update instead of a patch due to the breaking change?